types/autosize/tsconfig.json

```diff
@@ -6,18 +6,19 @@
             "dom"
         ],
         "noImplicitAny": true,
+        "noEmit": true,
```

Comment:
Library does not only export a namespace, but a function and object with same identifier `autosize`.
Also the lib doesn't look global it appears to be UMD module.
https://github.com/jackmoore/autosize/blob/master/dist/autosize.js

Regarding tsconfig, I didn't know and I will fix that at once we settle the other concern